### DISCUSSION TITLE
Add thumbnails for 3d print objects stl, 3mf

### DISCUSSION THREAD

**Discussion by mrspaceman** [spacedriveapp/spacedrive#2782] [Feature Request]
Add support for displaying thumbnails for 3d printing formats STL and 3MF.

- https://en.wikipedia.org/wiki/STL_(file_format)
- https://en.wikipedia.org/wiki/3D_Manufacturing_Format#:~:text=3MF%20is%20an%20XML%2Dbased,represented%20in%20the%20STL%20format.
- https://3mf.io/

**Comment by HeavenVolkoff** [comment]
Hello @mrspaceman, thanks for the suggestion. There is already a discussion for 3D objects thumbnail/preview, so I will close this in favor of the original one https://github.com/spacedriveapp/spacedrive/discussions/2454
